### ISSUE TITLE
Update UI to distinguish workspaces

### ISSUE THREAD

**Issue by AlmightyYakob** [multinet-app/multinet-client#117]
A list of workspaces can be broken up into three distinct categories (assuming a logged-in user): 
* Workspaces that are public
* Workspaces that you own
* Workspaces that you have permissions on

We need a way to split up the current entire list of workspaces that a user sees into a these three distinct categories. When logged out, the first case is the only option. When logged in, perhaps only the last 2 categories are worth actually separating, while the first one could just be denoted with an icon, since the first cast can overlap with the last two (i.e. any workspace that you own or have permissions on can also be public).

**Comment by AlmightyYakob** [comment]
@jtomeck What we're looking for here is an approach to the UI that you feel distinguishes the differences between these cases in the best way possible.

**Comment by JackWilb** [comment]
Just to reiterate some of our discussion from the slack channel, here are the things we had mentioned:

- Using tabs
- Using dividers to break up the horizonal space into three collapsible sections (public, owned, shared)

**Comment by waxlamp** [comment]
> Just to reiterate some of our discussion from the slack channel, here are the things we had mentioned:
> 
> * Using tabs
> * Using dividers to break up the horizonal space into three collapsible sections (public, owned, shared)
> * Using dividers to break up the horizonal space into two collapsible sections (yours, others) with icons denoting the ones that are public in the 'others' section

I prefer the idea of *two* sections, one for "my" workspaces, and one for other ones, possibly with a visual indicator for the ones that are public.

But this is exactly where we need @jtomeck's expertise.

**Comment by jtomeck** [comment]
I agree with @waxlamp on this one. If anyone's used Discord, I like how they split up users of different roles. I picture the list of workspaces could be split like that. Here's a screen of it as an example

<img width="238" alt="Screen Shot 2020-08-10 at 12 35 53 PM" src="https://user-images.githubusercontent.com/1560779/89807111-16fa2380-db06-11ea-80b1-23d74b450404.png">

**Comment by waxlamp** [comment]
And I am picturing a little blue badge that says `PUBLIC` for the public workspaces (almost exactly like the `PRO` badge looks on GitHub accounts such as that of AlmightyYakob).
